docs/content/manual/_incoming/2316-2384-deploy-guest-cluster-node-selector-label copy.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Deploy guest cluster to specific node with Node selector label
+---
+
+* Related issues: [#2316](https://github.com/harvester/harvester/issues/2316) [BUG] Guest cluster nodes distributed across failure domain
+* Related issues: [#2384](https://github.com/harvester/harvester/issues/2384) [backport v1.0.3] Guest cluster nodes distributed across failure domains
+
+## Category: 
+* Rancher integration
+
+## Verification Steps
+### RKE2 Verification Steps
+1. Open Harvester Host page then edit host config 
+1. Add the following key value in the labels page:
+  - topology.kubernetes.io/zone: zone_bp
+  - topology.kubernetes.io/region: region_bp
+  ![image](https://user-images.githubusercontent.com/29251855/179735384-77e99870-92ad-41c2-b414-a872130c0b27.png)
+1. Open the RKE2 provisioning page 
+1. Expand the show advanced 
+1. Click add Node selector in `Node scheduling`
+1. Use default `Required` priority
+1. Click Add Rule
+1. Provide the following key/value pairs
+  - `topology.kubernetes.io/zone: zone_bp`
+  - `topology.kubernetes.io/region: region_bp`
+  ![image](https://user-images.githubusercontent.com/29251855/179736419-78612fd1-9990-44d8-b9be-d9a850bd27a0.png)
+
+1. Provide the following user data 
+  ```
+  password: 123456
+  chpasswd: { expire: False }
+  ssh_pwauth: True
+  ```
+1. Create the RKE2 guest cluster
+1. Go to Harvester Virtual Machine page
+1. Edit yaml of the RKE2 guest cluster 
+  ![image](https://user-images.githubusercontent.com/29251855/179737541-f921d841-13ee-4a97-b096-5e4bdca39320.png)
+  Check the node affinity label have written into the yaml 
+  ![image](https://user-images.githubusercontent.com/29251855/179737712-9c0dac59-78be-4386-b1e4-646d7d7fbd90.png)
```

Comment:
image should have carriage return (newline) before it for easier formatting

Reply:
Thanks for the check, change all image content with newline follow with a tab indent (2 space)